### PR TITLE
Remove some uneeded patches.

### PR DESCRIPTION
**Pull request by crummel**

### CODE REVIEW

**Comment by crummel** [comment]
This gets rid of I believe all the ASP.NET patches we decided were questionable.  The baseline changes are expected because we're reverting to building on ASP.NET's expected target framework.

File: patches/aspnet-extensions/0004-Exclude-two-non-shipping-projects-from-source-build.patch [hunk 1]
```diff
@@ -12,7 +12,7 @@ index 0198da33a..d358b96e7 100644
 --- a/src/TargetingPack/Microsoft.Internal.Extensions.Refs/ref/Microsoft.Internal.Extensions.Refs.csproj
 +++ b/src/TargetingPack/Microsoft.Internal.Extensions.Refs/ref/Microsoft.Internal.Extensions.Refs.csproj
 @@ -4,6 +4,7 @@
-     <TargetFramework>netcoreapp2.0</TargetFramework>
+     <TargetFramework>netcoreapp3.0</TargetFramework>
```
Comment (JunTaoLuo) [inline patches/aspnet-extensions/0004-Exclude-two-non-shipping-projects-from-source-build.patch]: Hmm why is this change needed if we are excluding this from source build?

Comment (crummel) [reply patches/aspnet-extensions/0004-Exclude-two-non-shipping-projects-from-source-build.patch]: Sorry, this is just the confusing double-patch notation.  `netcoreapp3.0` is what you actually have, previously we were changing it to `netcoreapp2.0` and then disabling it, now we just disable it.
